packages/manager/modules/pci/src/projects/project/project.html

```diff
@@ -9,6 +9,8 @@
             data-user="$ctrl.user"
             data-is-open="$ctrl.isSidebarOpen"
             data-on-click="$ctrl.closeSidebar()"
+            data-go-to-project="$ctrl.goToProject"
```

Comment:
```suggestion
            data-go-to-project="$ctrl.goToProject()"
```
?

Reply:
We are just passing the method reference to the component. Hence, we are not calling it here